src/foglamp-service.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+### BEGIN INIT INFO
+# Provides:          foglamp
+# Required-Start:    $remote_fs $syslog
+# Required-Stop:     $remote_fs $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: foglamp script
+# Description:       starts foglamp core, storage and scheduler
+
+# `make install` will put foglamp as python code entry_point in /home/foglamp/.local/bin
+
+
+# Copy this script into /etc/init.d using
+# e.g. sudo cp foglamp-service.sh /etc/init.d.
+# Make sure the script is executable (chmod again)
+# sudo chmod 755 foglamp-service.sh
+# and make sure that it has UNIX line-endings (dos2unix).
+
+
+# running the command `sudo update-rc.d foglamp-service.sh defaults`.
+# This command adds in symbolic links to the `/etc/rc?.d` directories
+# so that the init script is run at the default times.
+# you can see these links if you do `ls -l /etc/rc?.d/*foglamp-service.sh`
+
+
+# At this point you should be able to start foglamp using the command:
+
+# sudo /etc/init.d/foglamp-service.sh start,
+# check its status with the /etc/init.d/foglamp-service.sh status argument
+# and stop it with sudo /etc/init.d/foglamp-service.sh stop.
+### END INIT INFO
+
+DIR=/home/foglamp/.local/bin
```

Comment:
Where has this directory come from?

Reply:
Installing Python packages to a system directory (as `/usr/local/lib/python3.5` and entry point to /usr/local/bin). This requires root access - (sudo) make install

`--user` makes pip / `python3 setup.py install` packages in user's home directory instead, which doesn't require any special privileges.


 https://github.com/foglamp/FogLAMP/pull/232/commits/5d9def6e2344ee5ff43b7189870bd7aa33a0b7d0, shows the difference

Reply:
Yes, it is well understood that make install needs root access to match the required structure.